src/registry/coverage.ts

```diff
@@ -27,10 +27,15 @@ type ApiVersion = {
   version: string;
 };
 
+let apiVer: number;
+
 export const getCurrentApiVersion = async (): Promise<number> => {
-  const apiVersionsUrl = 'https://dx-extended-coverage.my.salesforce-sites.com/services/data';
-  const lastVersionEntry = (await got(getProxiedOptions(apiVersionsUrl)).json<ApiVersion[]>()).pop() as ApiVersion;
-  return +lastVersionEntry.version;
+  if (apiVer === undefined) {
+    const apiVersionsUrl = 'https://appexchange.salesforce.com/services/data';
+    const lastVersionEntry = (await got(getProxiedOptions(apiVersionsUrl)).json<ApiVersion[]>()).pop() as ApiVersion;
+    apiVer = +lastVersionEntry.version;
+  }
+  return apiVer;
```

Comment:
more meta--if this *does* fail (network issues, org down, etc) would the error be meaningful for users to know what's happening?

it's 
```
Uncaught RequestError: getaddrinfo ENOTFOUND appexchange.salesforce.com
```

And I think it'd be nice to tell them why we were doing that and what they might do about it (ex: setting or passing in an api version).  It's hard for SDR to know what they're doing to be more specific.

Reply:
Internally, the error is never displayed to an end user because if it fails, the code in `ComponentSet` will just use api version 58.0.

Seems this method is exported though which means anyone could call it, and in that case this function has a misleading name and could cause other problems.  I would argue this should not be exported.